WikiPreview.Fluent.Plugin/WikiPreviewSearchResult.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections.ObjectModel;
+using Blast.Core.Interfaces;
+using Blast.Core.Results;
+using static WikiPreview.Fluent.Plugin.WikiPreviewSearchOperation;
+
+namespace WikiPreview.Fluent.Plugin
+{
+    public sealed class WikiPreviewSearchResult : SearchResultBase
+    {
+
+        public static readonly ObservableCollection<ISearchOperation> supportedOperations = new()
+        {
+            new WikiSearchOperation(),
+            new WikiwandSearchOperation(),
+            new GoogleSearchOperation(),
+            new CopyUrlSearchOperation()
+        };
+
+        public static readonly ObservableCollection<SearchTag> searchTags = new()
+        {
+            new SearchTag
+            { Name = WikiPreviewSearchApp.WikiSearchTagName, IconGlyph = "\uEDE4", Description = "Search in Wikipedia" },
+        };
+
+        public WikiPreviewSearchResult() : base()
+        {
+            PinUniqueId = PageID;
+            SearchObjectId = PageID;
+            Tags = searchTags;
+            SupportedOperations = supportedOperations;
+            ProcessInfo = null;
+
+            if (PreviewImage == null || PreviewImage.IsEmpty)
+                IconGlyph = "\uF6FA";
+        }
+
+        public string URL { get; set; }
```

Comment:
Why not move all URL initialization code here? the search app code is too long anyway

Reply:
All URL related declarations have been moved here. Please let me know on this.